### ISSUE TITLE
Chart datapoint interpolation/tension is funky

### ISSUE THREAD

**Issue by jamesinc** [mrvautin/adminMongo#192]
See this screenshot:

![](https://i.imgur.com/isZCBnY.png)

It seems like it's a known shortcoming of Chart.js's bezier function, but there are a couple of fixes written-up in the issue comments for charts where you have a known x-axis boundary (which as far as I've seen is always `y=0` for adminMongo's charts).

https://github.com/chartjs/Chart.js/issues/424#issuecomment-51399997

**Comment by makseq** [comment]
capBezierPoints doesn't work :(

**Comment by PropelNZ** [comment]
Setting tension: 0 took much of the aesthetic out so I edited the non-minified version of Chart.js v2.9.4

Find at at line 2498:
`		ctx.bezierCurveTo(
				flip ? previous.controlPointPreviousX : previous.controlPointNextX,
				flip ? previous.controlPointPreviousY : previous.controlPointNextY,
				flip ? target.controlPointNextX : target.controlPointPreviousX,
				flip ? target.controlPointNextY : target.controlPointPreviousY,
				target.x,
				target.y);`
Edit to:
`		if (previous.y == target.y) {
			ctx.lineTo(target.x, target.y);
		} else {
			ctx.bezierCurveTo(
				flip ? previous.controlPointPreviousX : previous.controlPointNextX,
				flip ? previous.controlPointPreviousY : previous.controlPointNextY,
				flip ? target.controlPointNextX : target.controlPointPreviousX,
				flip ? target.controlPointNextY : target.controlPointPreviousY,
				target.x,
				target.y);
        }`

Basically checks start and end points. If they are on the same y height, just draw a straight line.

Fixed my issue from this:
![bezier_default](https://user-images.githubusercontent.com/71102042/105146708-29709280-5b65-11eb-8f03-4a8de05e9ef7.JPG)

To this:
![bezier_flat](https://user-images.githubusercontent.com/71102042/105146745-342b2780-5b65-11eb-809a-6403c84d1036.JPG)
